index.html

```diff
@@ -12970,35 +12970,44 @@ <h2>A project for first-time contributions</h2>
 
             <!-- ________ Contributor's card START ________  -->
             <div class="card">
-                <p class="name">Name</p>
+                <p class="name">Rachitt Shah</p>
```

Comment:
It looks like you've modified the contributor's card template. You need to copy this template and paste it above the line
```
<!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
```
You can refer to the other cards in this file for reference or reach out to us if you need any help.